swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, category = 'Clothing', condition = 0):
+        self.category = category
+        self.condition = condition
```

Comment:
You can use inheritance here by invoking super()
```suggestion
            super().__init__(category="Clothing",condition)
```